css/styles.css

```diff
@@ -41,11 +41,13 @@
 /* @group Columns */
 
 .ab-wrap {
-    margin: 20px 0 0 -30px;
+    margin: 0 0 0 -10px;
+    padding: 20px 0 0 0;
+    text-rendering: optimizeLegibility;
```

Comment:
`text-rendering: optimizeLegibility;` was previously removed without actual reason. Since it affects browser behavior this should not be removed without further investigation.
See [developer.mozilla.org](https://developer.mozilla.org/en/docs/Web/CSS/text-rendering)